### DISCUSSION TITLE
馒头站点的添加或者连接失败

### DISCUSSION THREAD

**Discussion by pzxbc** [jxxghp/MoviePilot#2589] [分享]
馒头站点的站点地址一定要填https://api.m-team.cc， 而不是之前的https://kp.m-team.cc。

因为站点现在停止了之前域名调用api的方式，只能使用api对应的域名地址。另外cookie也不需要了，之前有填过的话，手动删除一下

**Comment by HeatonZ** [comment]
也是连不上, 测试不通过
看了代码提示, 运行的代码的可能是通用测试

**Comment by jacky940810** [comment]
我因为开了二次验证没办法连上，但是又没有地方能取消

**Comment by fanyunxu** [comment]
感谢分享   使用[[](https://api.m-team.cc,/)](https://api.m-team.cc) 这个域名成功了

**Comment by pzxbc** [comment]
原来还有不少人遇到这问题😅。可惜我账号无冤被封了，不能再囤纪录片了，有大佬邀请下么？

**Comment by Stong2** [comment]
我也是用API无法登陆，确实开了二级验证，请问现在有什么解决办法吗

**Reply by henryyan** [reply]
同问，配置了 auth 和 api key 提示无法认证

**Reply by Stong2** [reply]
> 我也是用API无法登陆，确实开了二级验证，请问现在有什么解决办法吗

使用IYUU添加馒头账号后，将IYUU信息添加到MP；
再用MP添加馒头就可以了

**Reply by Gleiphir2769** [reply]
> > 我也是用API无法登陆，确实开了二级验证，请问现在有什么解决办法吗
> 
> 使用IYUU添加馒头账号后，将IYUU信息添加到MP； 再用MP添加馒头就可以了

求问，这个方法是针对开启了二级验证的账号还是目前只能这样连接，我发现现在我没开启二级验证也连不上了

**Reply by cddjr** [reply]
> > > 我也是用API无法登陆，确实开了二级验证，请问现在有什么解决办法吗
> 
> > 
> 
> > 使用IYUU添加馒头账号后，将IYUU信息添加到MP； 再用MP添加馒头就可以了
> 
> 
> 
> 求问，这个方法是针对开启了二级验证的账号还是目前只能这样连接，我发现现在我没开启二级验证也连不上了

连通测试失败无所谓，你浏览资源能看到种子列表就行了。

**Reply by waveblog** [reply]
@cddjr 想问下能看到种子列表下载不了种子是什么原因呀 显示无法获取下载地址
